### ISSUE TITLE
Pride Logo (+ Custom Logos?)

### ISSUE THREAD

**Issue by Scartlett** [eramdam/BetterTweetDeck#366]
Hey! I'm a huge fan of Pride, I love that it's a great way to share our actual selves with others that may have had a similar experience, regardless of this through - the London Pride month is over and that means the logo for TweetDeck has gone too.

I'm hoping it'd be a possibility for anyone to not only to toggle the pride logo back on (I have the .svg files, if needed) but also to change that area a up a little if that's desired by others, but I'm not quite sure on how that'll work.

Regardless, thank you for the fantastic extension and I can't wait to see the future of BTD!

**Comment by richardbmx** [comment]
@Scartlett  Hello my name is Richard I am a graphic designer and I love making contributions for open source projects, I see that you do not have a defined logo for your application. If you want I can do it for you without any problem you just have to authorize me to do it ... regards. ..

**Comment by NuroDev** [comment]
There is currently no way to change the logo back to the pride logo as of yet. That may come in future potentially.
However in the mean time if you do want the logo that badly then you can paste the code below into your custom CSS Tab in BTD options and it will override BTD log with the Tweetdeck Pride Logo.
```

.btd__replace_logo .application .app-title .visible-in-contracted-header .tweetdeck-logo {
  background-image: url('data:image/svg+xml;utf8,<svg width="26px" height="26px" viewBox="0 0 26 26" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g id="tweetdeck-beta-pride.a8d73430e8" fill-rule="nonzero"> <path d="M0.2,3.4 L25.2,3.4 C25.3,1.6 23.7,0.1 21.9,0.1 L3.6,0.1 C1.8,0.1 0.3,1.5 0.2,3.4 Z" id="Shape" fill="#E31F5D"></path> <polygon id="Shape" fill="#FF8D50" points="0.2 3.5 0.2 6.6 25.2 6.6 25.2 3.3 0.2 3.3"></polygon> <polygon id="Shape" fill="#FFD22A" points="0.2 6.5 25.2 6.5 25.2 9.9 0.2 9.9"></polygon> <polygon id="Shape" fill="#FFE765" points="0.2 9.8 25.2 9.8 25.2 13.1 0.2 13.1"></polygon> <polygon id="Shape" fill="#4DBA9B" points="0.2 13 25.2 13 25.2 16.3 0.2 16.3"></polygon> <polygon id="Shape" fill="#6DC8F9" points="0.2 16.2 0.2 19.5 25.2 19.5 25.2 16.2"></polygon> <path d="M0.2,19.4 C0.4,21.1 1.8,22.4 3.6,22.4 L9.3,22.4 L9.5,22.6 L15.9,22.6 L16.1,22.4 L21.8,22.4 C23.6,22.4 25,21.1 25.2,19.4 L0.2,19.4 Z" id="Shape" fill="#95E3FF"></path> <polygon id="Shape" fill="#C8B2FD" points="16 22.5 9.4 22.5 12.7 25.8"></polygon> <path d="M9.7,18.6 C16,18.6 19.5,13.4 19.5,8.8 L19.5,8.4 C20.2,7.9 20.8,7.3 21.2,6.6 C20.6,6.9 19.9,7.1 19.2,7.1 C19.9,6.7 20.5,6 20.7,5.2 C20,5.6 19.3,5.9 18.5,6 C17.9,5.3 17,4.9 16,4.9 C14.1,4.9 12.6,6.4 12.6,8.3 C12.6,8.6 12.6,8.8 12.7,9.1 C9.8,9 7.3,7.6 5.6,5.5 C5.3,6.1 5.1,6.6 5.1,7.3 C5.1,8.5 5.7,9.5 6.6,10.2 C6,10.2 5.5,10 5,9.8 C5,11.5 6.2,12.9 7.8,13.2 C7.5,13.3 7.2,13.3 6.9,13.3 C6.7,13.3 6.5,13.3 6.3,13.2 C6.7,14.6 8,15.6 9.5,15.6 C8.3,16.5 6.8,17.1 5.2,17.1 L4.4,17.1 C5.8,18 7.7,18.6 9.7,18.6 Z" id="Path" fill="#FFFFFF"></path> <path d="M20.4,15.3 C20.5,15 21,15 21.2,15.3 L21.7,16.8 L23.3,17.6 C23.6,17.8 23.6,18.2 23.2,18.3 L21.6,18.8 L20.9,20.6 C20.8,20.9 20.2,20.9 20.1,20.6 L19.7,18.8 L18.1,18.2 C17.8,18.1 17.8,17.6 18.1,17.5 L19.8,16.8 L20.4,15.3 Z" id="Shape" fill="#FFB6C1"></path> <path d="M20.8,15 C20.6,15 20.4,15.1 20.3,15.3 L19.7,16.7 L18.1,17.4 C17.7,17.5 17.7,18.2 18.1,18.3 L19.6,18.9 L20,20.6 C20,20.7 20.1,20.8 20.2,20.8 C20.3,20.8 20.4,20.9 20.5,20.9 C20.6,20.9 20.7,20.9 20.8,20.8 C20.9,20.8 21,20.7 21,20.6 L21.7,18.9 L23.2,18.4 C23.4,18.3 23.6,18.2 23.6,18 C23.6,17.8 23.5,17.6 23.3,17.5 L21.7,16.7 L21.2,15.2 C21.2,15 21,15 20.8,15 Z M20.8,15.2 C20.9,15.2 21.1,15.3 21.1,15.4 L21.6,16.9 L23.2,17.7 C23.3,17.8 23.4,17.9 23.4,18 C23.4,18.1 23.3,18.2 23.2,18.2 L21.5,18.7 L20.8,20.6 C20.8,20.6 20.8,20.7 20.7,20.7 L20.3,20.7 C20.2,20.7 20.2,20.6 20.2,20.6 L19.8,18.7 L18.1,18.1 C17.9,18 17.9,17.7 18.1,17.6 L19.9,16.9 L20.5,15.3 C20.5,15.2 20.6,15.2 20.8,15.2 Z" id="Shape" fill="#FFFFFF"></path> <path d="M3.5,1.5 C3.6,1.3 3.9,1.3 3.9,1.5 L4.3,2.6 L5.5,3.2 C5.7,3.3 5.7,3.5 5.5,3.6 L4.3,4 L3.8,5.3 C3.7,5.5 3.4,5.5 3.4,5.3 L3,4 L1.9,3.6 C1.7,3.5 1.7,3.3 1.9,3.2 L3.1,2.7 L3.5,1.5 Z" id="Shape" fill="#94E5FE"></path> <path d="M3.7,1.2 C3.6,1.2 3.4,1.3 3.4,1.4 L3,2.6 L1.8,3.1 C1.7,3.2 1.6,3.3 1.6,3.4 C1.6,3.5 1.7,3.6 1.8,3.7 L2.9,4.1 L3.3,5.3 C3.3,5.4 3.3,5.5 3.4,5.5 C3.5,5.5 3.5,5.6 3.6,5.6 C3.7,5.6 3.9,5.5 3.9,5.4 L4.4,4.1 L5.6,3.7 C5.7,3.6 5.8,3.5 5.8,3.4 C5.8,3.3 5.7,3.2 5.6,3.1 L4.4,2.5 L4,1.5 C4,1.4 4,1.3 3.9,1.3 C3.9,1.2 3.8,1.2 3.7,1.2 Z M3.7,1.5 C3.8,1.5 3.8,1.5 3.7,1.5 C3.8,1.5 3.8,1.5 3.7,1.5 L4.2,2.7 L5.4,3.3 C5.5,3.3 5.5,3.4 5.5,3.4 C5.5,3.4 5.5,3.5 5.4,3.5 L4.2,3.9 L3.7,5.2 C3.7,5.3 3.6,5.3 3.6,5.3 L3.1,3.9 L2,3.5 C1.9,3.5 1.9,3.4 1.9,3.4 C1.9,3.4 1.9,3.3 2,3.3 L3.2,2.8 L3.6,1.6 C3.6,1.5 3.7,1.5 3.7,1.5 Z" id="Shape" fill="#FFFFFF"></path> </g> </g> </svg>')!important;
}

.btd__replace_logo .application .app-title .invisible-in-contracted-header {
  background-image: url('data:image/svg+xml;utf8,<svg width="132px" height="26px" viewBox="0 0 132 26" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g id="tweetdeck-beta-pride.a8d73430e8" fill-rule="nonzero"> <path d="M0.2,3.4 L25.2,3.4 C25.3,1.6 23.7,0.1 21.9,0.1 L3.6,0.1 C1.8,0.1 0.3,1.5 0.2,3.4 Z" id="Shape" fill="#E31F5D"></path> <polygon id="Shape" fill="#FF8D50" points="0.2 3.5 0.2 6.6 25.2 6.6 25.2 3.3 0.2 3.3"></polygon> <polygon id="Shape" fill="#FFD22A" points="0.2 6.5 25.2 6.5 25.2 9.9 0.2 9.9"></polygon> <polygon id="Shape" fill="#FFE765" points="0.2 9.8 25.2 9.8 25.2 13.1 0.2 13.1"></polygon> <polygon id="Shape" fill="#4DBA9B" points="0.2 13 25.2 13 25.2 16.3 0.2 16.3"></polygon> <polygon id="Shape" fill="#6DC8F9" points="0.2 16.2 0.2 19.5 25.2 19.5 25.2 16.2"></polygon> <path d="M0.2,19.4 C0.4,21.1 1.8,22.4 3.6,22.4 L9.3,22.4 L9.5,22.6 L15.9,22.6 L16.1,22.4 L21.8,22.4 C23.6,22.4 25,21.1 25.2,19.4 L0.2,19.4 Z" id="Shape" fill="#95E3FF"></path> <polygon id="Shape" fill="#C8B2FD" points="16 22.5 9.4 22.5 12.7 25.8"></polygon> <path d="M36.1,4.9 L36.1,7.4 L40.1,7.4 L40.1,18.4 L43.1,18.4 L43.1,7.4 L47.2,7.4 L47.2,4.9 L36.1,4.9 Z M87.8,4.9 L87.8,18.4 L93.6,18.4 C97.7,18.4 99.7,15.5 99.7,11.6 C99.7,7.1 97.1,4.9 93.6,4.9 L87.8,4.9 Z M121.4,4.9 L121.4,18.5 L124.1,18.5 L124.1,15.1 L125.1,14.1 L127.8,18.5 L131.1,18.5 L127,12.3 L130.7,8.7 L127.5,8.7 L124.1,12.2 L124.1,4.9 L121.4,4.9 Z M82,5.7 L82,8.7 L80.3,8.7 L80.3,10.5 L81.9,10.5 L81.9,16.3 C81.9,18.2 83.3,18.6 85,18.6 C85.6,18.6 86.1,18.6 86.6,18.5 L86.6,16.4 C86.4,16.5 86.1,16.5 85.8,16.5 C84.9,16.5 84.7,16.3 84.7,15.4 L84.7,10.5 L86.7,10.5 L86.7,8.7 L84.7,8.7 L84.7,5.7 L82,5.7 Z M90.9,7.4 L93,7.4 C95.9,7.4 96.9,9.2 96.9,11.9 C96.9,14.9 95.2,16 93.5,16 L90.9,16 L90.9,7.4 Z M65.1,8.3 C62.1,8.3 60.1,10.6 60.1,13.5 C60.1,16.6 62,18.7 65.1,18.7 C67.3,18.7 68.9,17.7 69.7,15.4 L67.3,15.4 C67.2,16 66.3,16.7 65.2,16.7 C63.7,16.7 62.9,15.9 62.8,14.2 L62.8,14.1 L69.9,14.1 C70.1,11.1 68.4,8.3 65.1,8.3 Z M75.6,8.3 C72.6,8.3 70.6,10.6 70.6,13.5 C70.6,16.6 72.5,18.7 75.6,18.7 C77.8,18.7 79.4,17.7 80.2,15.4 L77.8,15.4 C77.7,16 76.8,16.7 75.7,16.7 C74.2,16.7 73.4,15.9 73.3,14.2 L73.3,14.1 L80.4,14.1 C80.6,11.1 78.9,8.3 75.6,8.3 Z M105.5,8.3 C102.5,8.3 100.5,10.6 100.5,13.5 C100.5,16.6 102.4,18.7 105.5,18.7 C107.7,18.7 109.3,17.7 110.1,15.4 L107.7,15.4 C107.6,16 106.7,16.7 105.6,16.7 C104.1,16.7 103.3,15.9 103.2,14.2 L103.2,14.1 L110.3,14.1 C110.5,11.1 108.8,8.3 105.5,8.3 Z M115.8,8.4 C112.6,8.4 110.8,10.6 110.8,13.7 C110.8,16.6 112.7,18.7 115.7,18.7 C118.3,18.7 120,17.3 120.4,14.7 L117.8,14.7 C117.7,15.9 117,16.7 115.7,16.7 C114.1,16.7 113.5,15 113.5,13.6 C113.5,12.1 114.1,10.4 115.8,10.4 C117,10.4 117.6,11 117.8,12.1 L120.4,12.1 C120.3,9.6 118.1,8.4 115.8,8.4 Z M45.2,8.7 L48.3,18.5 L51.1,18.5 L52.8,11.9 L54.5,18.5 L57.3,18.5 L60.3,8.7 L57.5,8.7 L55.7,15.3 L54,8.7 L51.4,8.7 L49.8,15.4 L48,8.7 L45.2,8.7 Z M65.1,10.4 C66.4,10.4 67,11.1 67.2,12.5 L62.9,12.5 C62.9,11.7 63.4,10.4 65.1,10.4 Z M75.6,10.4 C76.9,10.4 77.5,11.1 77.7,12.5 L73.4,12.5 C73.4,11.7 73.9,10.4 75.6,10.4 Z M105.4,10.4 C106.7,10.4 107.3,11.1 107.5,12.5 L103.2,12.5 C103.2,11.7 103.7,10.4 105.4,10.4 Z" id="Shape" fill="#FFFFFF"></path> <path d="M9.7,18.6 C16,18.6 19.5,13.4 19.5,8.8 L19.5,8.4 C20.2,7.9 20.8,7.3 21.2,6.6 C20.6,6.9 19.9,7.1 19.2,7.1 C19.9,6.7 20.5,6 20.7,5.2 C20,5.6 19.3,5.9 18.5,6 C17.9,5.3 17,4.9 16,4.9 C14.1,4.9 12.6,6.4 12.6,8.3 C12.6,8.6 12.6,8.8 12.7,9.1 C9.8,9 7.3,7.6 5.6,5.5 C5.3,6.1 5.1,6.6 5.1,7.3 C5.1,8.5 5.7,9.5 6.6,10.2 C6,10.2 5.5,10 5,9.8 C5,11.5 6.2,12.9 7.8,13.2 C7.5,13.3 7.2,13.3 6.9,13.3 C6.7,13.3 6.5,13.3 6.3,13.2 C6.7,14.6 8,15.6 9.5,15.6 C8.3,16.5 6.8,17.1 5.2,17.1 L4.4,17.1 C5.8,18 7.7,18.6 9.7,18.6 Z" id="Path" fill="#FFFFFF"></path> <path d="M20.4,15.3 C20.5,15 21,15 21.2,15.3 L21.7,16.8 L23.3,17.6 C23.6,17.8 23.6,18.2 23.2,18.3 L21.6,18.8 L20.9,20.6 C20.8,20.9 20.2,20.9 20.1,20.6 L19.7,18.8 L18.1,18.2 C17.8,18.1 17.8,17.6 18.1,17.5 L19.8,16.8 L20.4,15.3 Z" id="Shape" fill="#FFB6C1"></path> <path d="M20.8,15 C20.6,15 20.4,15.1 20.3,15.3 L19.7,16.7 L18.1,17.4 C17.7,17.5 17.7,18.2 18.1,18.3 L19.6,18.9 L20,20.6 C20,20.7 20.1,20.8 20.2,20.8 C20.3,20.8 20.4,20.9 20.5,20.9 C20.6,20.9 20.7,20.9 20.8,20.8 C20.9,20.8 21,20.7 21,20.6 L21.7,18.9 L23.2,18.4 C23.4,18.3 23.6,18.2 23.6,18 C23.6,17.8 23.5,17.6 23.3,17.5 L21.7,16.7 L21.2,15.2 C21.2,15 21,15 20.8,15 Z M20.8,15.2 C20.9,15.2 21.1,15.3 21.1,15.4 L21.6,16.9 L23.2,17.7 C23.3,17.8 23.4,17.9 23.4,18 C23.4,18.1 23.3,18.2 23.2,18.2 L21.5,18.7 L20.8,20.6 C20.8,20.6 20.8,20.7 20.7,20.7 L20.3,20.7 C20.2,20.7 20.2,20.6 20.2,20.6 L19.8,18.7 L18.1,18.1 C17.9,18 17.9,17.7 18.1,17.6 L19.9,16.9 L20.5,15.3 C20.5,15.2 20.6,15.2 20.8,15.2 Z" id="Shape" fill="#FFFFFF"></path> <path d="M3.5,1.5 C3.6,1.3 3.9,1.3 3.9,1.5 L4.3,2.6 L5.5,3.2 C5.7,3.3 5.7,3.5 5.5,3.6 L4.3,4 L3.8,5.3 C3.7,5.5 3.4,5.5 3.4,5.3 L3,4 L1.9,3.6 C1.7,3.5 1.7,3.3 1.9,3.2 L3.1,2.7 L3.5,1.5 Z" id="Shape" fill="#94E5FE"></path> <path d="M3.7,1.2 C3.6,1.2 3.4,1.3 3.4,1.4 L3,2.6 L1.8,3.1 C1.7,3.2 1.6,3.3 1.6,3.4 C1.6,3.5 1.7,3.6 1.8,3.7 L2.9,4.1 L3.3,5.3 C3.3,5.4 3.3,5.5 3.4,5.5 C3.5,5.5 3.5,5.6 3.6,5.6 C3.7,5.6 3.9,5.5 3.9,5.4 L4.4,4.1 L5.6,3.7 C5.7,3.6 5.8,3.5 5.8,3.4 C5.8,3.3 5.7,3.2 5.6,3.1 L4.4,2.5 L4,1.5 C4,1.4 4,1.3 3.9,1.3 C3.9,1.2 3.8,1.2 3.7,1.2 Z M3.7,1.5 C3.8,1.5 3.8,1.5 3.7,1.5 C3.8,1.5 3.8,1.5 3.7,1.5 L4.2,2.7 L5.4,3.3 C5.5,3.3 5.5,3.4 5.5,3.4 C5.5,3.4 5.5,3.5 5.4,3.5 L4.2,3.9 L3.7,5.2 C3.7,5.3 3.6,5.3 3.6,5.3 L3.1,3.9 L2,3.5 C1.9,3.5 1.9,3.4 1.9,3.4 C1.9,3.4 1.9,3.3 2,3.3 L3.2,2.8 L3.6,1.6 C3.6,1.5 3.7,1.5 3.7,1.5 Z" id="Shape" fill="#FFFFFF"></path> </g> </g></svg>')!important;
}
```

**Comment by Scartlett** [comment]
@richardbmx - I don't quite understand what you mean, sorry - have a good day though!

@NuroDev - thank you so much, however it doesn't seem to have any effect. Of course I have the general warning icons in the CSS editor, however, the pride logo has been disabled or something with the CSS (text) - sorry if I happen to be making a mistake, but I don't think I am.

**Comment by Scartlett** [comment]
Is there another possible CSS I could use?

**Comment by eramdam** [comment]
Been working on this for the past few days, should be in the next release, whenever I can get it out 😅 
![image](https://user-images.githubusercontent.com/1409924/120707332-09d06b80-c46f-11eb-8e7d-dd22f185e338.png)

**Comment by eramdam** [comment]
Fixed in [4.2.0](https://github.com/eramdam/BetterTweetDeck/releases/tag/4.2.0)
